sru-scripts/manual/ec2-sru

```diff
@@ -0,0 +1,146 @@
+#!/bin/sh
+
+set -ex
+
+SRU_VARS=./sru-scripts/sru-vars.template
+
+if [ ! -f $SRU_VARS ]; then
+   echo "Missing config environment file $SRU_VARS."
+   exit 1
+fi
+
+. $SRU_VARS
+
+parse_args $0 $@
+assert_expected_vars PROPOSED_SCRIPT LP_USER SAMPLE_CLOUDCONFIG SSHOPTS \
+    PRESERVE_INSTANCE
+create_samplecloudconfig
```

Comment:
I know this is not related to this PR, but looking at `create_samplecloudconfig`, it seems that the `filename` variable is not being used in the function:

https://github.com/cloud-init/ubuntu-sru/blob/master/sru-scripts/sru-vars.template#L45

Reply:
good point, I fell back to use SAMPLE_CLOUDCONFIG if filename param wasn't specified with 
`filename=${1:-${SAMPLE_CLOUDCONFIG}}`
but better to be explicit. I expect the function will have to change shortly anyway if anyone grows a slightly more customized need for it.